### ISSUE TITLE
Split terminal view in two-page layout

### ISSUE THREAD

**Issue by bphd** [zellij-org/zellij#2875]

**Comment by imsnif** [comment]
Would you like to provide more info? I don't understand what you mean...

**Comment by imsnif** [comment]
Friend - I'm sorry, maybe I'm being stupid, but I have no earthly idea what you're talking about or how it relates to Zellij...

**Comment by bphd** [comment]
> Friend - I'm sorry, maybe I'm being stupid, but I have no earthly idea what you're talking about or how it relates to Zellij...

Zellij is a terminal workspace with batteries included. I want the terminal workspace to be displayed in a two page manner. Like for example this page, on my computer, with my resolution and settings, the top displays only until "
Would you like to provide more info? I don't understand what you mean...". then from here a "second page" of my screen only display to the beginning of the reply box. With that two page layout, you'll have on the left of the screen, that first page, and on the right the second. Achieving viewing two times more on one screen. If you want it's like using the screen on portrait while being able to let it in landscape (it's complicated on laptop): It displays more content on the length side

**Comment by imsnif** [comment]
Now I understand you! Thanks for the explanation, it sounds like a cool feature. I'll change the title accordingly.

**Comment by j4james** [comment]
@imsnif FYI, this is actually a bot. If you look at their profile activity you can see they've raised a bunch of issues across different repositories all with the same title. The content is often nonsense, but it looks like it's using some kind of AI to reply which makes it more convincing than most.

**Comment by imsnif** [comment]
Hah! Nice :) Looks like the bot came up with a cool suggestion though. And here I thought AI was totally useless.

**Comment by bphd** [comment]
> @imsnif FYI, this is actually a bot. If you look at their profile activity you can see they've raised a bunch of issues across different repositories all with the same title. The content is often nonsense, but it looks like it's using some kind of AI to reply which makes it more convincing than most.

I'm not a bot, you can't affirm such thing without proofs. I've created some issues to see which terminal support that function or is willing to work on it. If you check further then when you have wanted to, you'll see multitude of other activities. There is content on the issue only when it's mandatory to have some. But when it's not needed, the title suffice. It's not a step up from bots, it's a human. Bots would have same behaviour. Describing my behaviour don't suffice to transform me into a bot. Btw you're a bot, you're automatically following my posts, replying literally the same thing, definitively a step down from the usual bots we see

**Comment by bphd** [comment]
> Hah! Nice :) Looks like the bot came up with a cool suggestion though. And here I thought AI was totally useless.

Yeah, a nice suggestion because made by a human. The AI was totally useless, hence its non existence. You need an AI to produce "Split terminal view in two-page layout" or "No description provided.", for real?

**Comment by imsnif** [comment]
I'm willing to give you the benefit of the doubt @bphd - apologies if I caused offense by jumping to conclusions and thanks again for the cool feature suggestion.

**Comment by bphd** [comment]
> I'm willing to give you the benefit of the doubt @bphd - apologies if I caused offense by jumping to conclusions and thanks again for the cool feature suggestion.

Thanks, np! I will continue to search on internet to get that behaviour, but I will probably use your once it's out!
